go/arrow/array/struct.go

```diff
@@ -81,6 +81,15 @@ func NewStructData(data arrow.ArrayData) *Struct {
 func (a *Struct) NumField() int           { return len(a.fields) }
 func (a *Struct) Field(i int) arrow.Array { return a.fields[i] }
 
+func (a *Struct) ValueStr(i int) string {
+	var buf bytes.Buffer
+	enc := json.NewEncoder(&buf)
+	if err := enc.Encode(a.GetOneForMarshal(i)); err != nil {
+		return ""
```

Comment:
we shouldn't swallow the error. either return `err.String()` or `panic(err)` please